src/core/qgswebview.h

```diff
@@ -41,9 +41,9 @@ class CORE_EXPORT QgsWebView : public QWebView
     {
       QDesktopWidget desktop;
       // Apply zoom factor for HiDPI screens
-      if ( desktop.physicalDpiX() > 96 )
+      if ( desktop.logicalDpiX() > 96 )
```

Comment:
What does this change do?
For a related discussion see https://github.com/qgis/QGIS/pull/9310

Reply:
The `logicalDpiX` appears to be the correct choice here because it's user-settable (depending on, say, the distance from your eyes etc.). 

I tested it on two different HiDPI setups where the current implementation which uses `physicalDpi` had no practical effect (and text was way too small).

Can you recommend a better approach?

Reply:
It's probably the correct thing to use logical dpi here (where we don't do pixel rendering as in the map canvas). Just wasn't sure about it being packaged in a different pull request with no mentioning about reasoning in the description. But looks good to me.

Reply:
>  Just wasn't sure about it being packaged in a different pull request with no mentioning about reasoning in the description. 

Oh, those lazy programmers ;)

Reply:
Nah, it's those control freaks doing the reviews ;)